mavros/include/mavros/setpoint_mixin.h

```diff
@@ -162,8 +166,10 @@ class SetAttitudeTargetMixin {
 template <class D>
 class TF2ListenerMixin {
 public:
-	std::thread tf_thread;
-	std::string tf_thd_name;
+	std::thread tf_thread_1;
+	std::thread tf_thread_2;
+	std::string tf_thd_name_1;
+	std::string tf_thd_name_2;
```

Comment:
If you are using one thread for `tf2_start` in one plugin and another `tf2_start` for another plugin which is using sync (the case of `setpoint_ attitude`), shouldn't two different threads be used?